pages/market.js

```diff
@@ -101,6 +103,12 @@ function MarketPage({ serverQuery }) {
 		setIsFetching(false)
 	}
 
+	const onClickDisplay = (typeDisplay) => {
+		setTokens([])
+		_fetchData()
+		setDisplay(typeDisplay)
+	}
+
```

Comment:
since ini ngeset tokensnya jadi empty array lagi, berarti ngefetch dari 0. `idNext`, `lowestPriceNext`, `updatedAtNext` sama `hasMore` nitip di update juga ya ke initial valuenya